actions/xmpp

```diff
@@ -230,6 +234,67 @@ def subcommand_change_domainname(arguments):
     action_utils.service_start('ejabberd')
 
 
+def subcommand_add_letsencrypt(arguments):
+    """Start using letsencrypt certificate if it matches configured domain."""
+    with open(EJABBERD_CONFIG, 'r') as file_handle:
+        conf = ruamel.yaml.round_trip_load(file_handle, preserve_quotes=True)
+
+    if arguments.domain in conf['hosts']:
+        cert_folder = '/etc/ejabberd/letsencrypt/' + arguments.domain
+        if not os.path.exists(cert_folder):
+            os.makedirs(cert_folder)
+            shutil.chown(cert_folder, 'ejabberd', 'ejabberd')
+
+        cert_file = cert_folder + '/ejabberd.pem'
+        le_folder = '/etc/letsencrypt/live/' + arguments.domain
+        with open(cert_file, 'w') as outfile:
+            with open(le_folder + '/privkey.pem', 'r') as infile:
+                for line in infile:
+                    if line.strip():
+                        outfile.write(line)
+            with open(le_folder + '/fullchain.pem', 'r') as infile:
+                for line in infile:
+                    if line.strip():
+                        outfile.write(line)
```

Comment:
One issue is that ejabberd requires the cert and key to be combined in a single pem file (https://github.com/processone/ejabberd/issues/826). But certbot doesn't support outputting this format yet (https://github.com/certbot/certbot/issues/1201).

Another issue is that the LE certs are only readable by root (https://github.com/certbot/certbot/issues/1425).

If those 2 certbot issues are fixed, then we shouldn't need to make a copy of the cert. So we may want to hold off on merging this PR until then (or we will need to migrate and clean up current user settings later).